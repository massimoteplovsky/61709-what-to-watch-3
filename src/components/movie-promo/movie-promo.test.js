import React from 'react';
import renderer from 'react-test-renderer';
import MoviePromo from './movie-promo.jsx';

it(`<MoviePromo /> component renders correctly`, () => {
  const tree = renderer
    .create(<MoviePromo
      title={``}
      genre={``}
      year={2014}
    />, {
      createNodeMock: () => {
        return {};
      }}).toJSON();
  expect(tree).toMatchSnapshot();
});
