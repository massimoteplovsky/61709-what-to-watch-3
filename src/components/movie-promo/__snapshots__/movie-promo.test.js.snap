// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MoviePromo /> component renders correctly 1`] = `
<div
  className="movie-card__wrap"
>
  <div
    className="movie-card__desc"
  >
    <h2
      className="movie-card__title"
    >
      Film 3
    </h2>
    <p
      className="movie-card__meta"
    >
      <span
        className="movie-card__genre"
      >
        Drama
      </span>
      <span
        className="movie-card__year"
      >
        2019
      </span>
    </p>
    <div
      className="movie-card__buttons"
    >
      <a
        className="btn btn--play movie-card__button"
        href="/player/1"
        onClick={[Function]}
        type="button"
      >
        <svg
          height="19"
          viewBox="0 0 19 19"
          width="19"
        >
          <use
            xlinkHref="#play-s"
          />
        </svg>
        <span>
          Play
        </span>
      </a>
      <button
        className="btn btn--list movie-card__button"
        onClick={[Function]}
        type="button"
      >
        <svg
          height="14"
          viewBox="0 0 18 14"
          width="18"
        >
          <use
            xlinkHref="#in-list"
          />
        </svg>
        <span>
          My list
        </span>
      </button>
      <a
        className="btn movie-card__button"
        href="add-review.html"
      >
        Add review
      </a>
    </div>
  </div>
</div>
`;
